src/cache.js

```diff
@@ -69,6 +69,25 @@ query getProjectsV2Fields($id: ID!, $cursor: String!) {
 	}
 }
 `;
+const GetRepositoryDescription = gql`
+query GetRepositoryDescription($orgName: String!, $repoName: String!){
+    repository(owner: $orgName, name: $repoName) {
+		description
+        main: object(expression: "main:README.md") {
+            ... on Blob {
+                text
+            }
+        }
+        master: object(expression: "master:README.md") {
+            ... on Blob {
+                text
+            }
+        }        
+    }
+}
+`; //Loophole to get the ReadMe No Matter the branch name
+
+const serviceApps = ["account","nex","friends","wii u chat", "juxt", "website"]
```

Comment:
```suggestion
const serviceApps = ["account", "nex", "friends", "wii u chat",  "juxt",  "website"];
```

This could also probably go on multiple lines, but I won't require it